scripts/actions/mobskills/forceful_blow.lua

```diff
@@ -10,14 +10,18 @@
 local mobskillObject = {}
 
 mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    if mob:getAnimationSub() == 0 then
+        return 1
+    end
+
     return 0
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
     local numhits = 1
     local accmod = 1
-    local dmgmod = 3
-    local info = xi.mobskills.mobPhysicalMove(mob, target, skill, numhits, accmod, dmgmod, xi.mobskills.magicalTpBonus.NO_EFFECT)
+    local dmgmod = 1
+    local info = xi.mobskills.mobPhysicalMove(mob, target, skill, numhits, accmod, dmgmod, xi.mobskills.physicalTpBonus.DMG_VARIES, 1, 1.5, 2)
```

Comment:
Funny enough `xi.mobskills.physicalTpBonus.DMG_VARIES` doesnt actually do anything. It enters this:
```lua
    if tpEffect == xi.mobskills.physicalTpBonus.DMG_VARIES then
        hitdamage = hitdamage * MobTPMod(skill:getTP() / 10)
    end
```
and then since even at 3000 tp the function `MobTPMod` just returns 1.
I want to say the "Damage varies with TP" is just the FTP values you provided at the end.

Reply:
I looked into some of these functions and there are many issues with them. I've started to wrap my head around how this is all suppose to work but there is still much I'm piecing together. Correcting this is a whole project in itself.